lib/ansible/plugins/filter/ternary.yml

```diff
@@ -0,0 +1,47 @@
+DOCUMENTATION:
```

Comment:
Does validate-modules check these? We should ensure that's added in the same release otherwise we will soon end up with badly formatted DOCUMENTATION blocks.

Reply:
always part of the plan, but not worth starting until we narrow down the new structure

Reply:
Also general support of plugins for validate-modules should be in place first :)